params/params_privnet.go

```diff
@@ -134,5 +135,16 @@ var PrivNetParams = Params{
 	// TODO replace the test pkh
 	//OrganizationPkScript:  hexMustDecode("76a91408ff3106060bf8d7d61a25d8108ec977698729f788ac"),
 
+	// Because it's only for testing, it comes from testwallet.go
+	TokenAdminPkScript: hexMustDecode("000000c9756d76a914785bfbf4ecad8b72f2582be83616c5d364a3244288ac"),
+
 	CoinbaseMaturity: 16,
+
```

Comment:
I think we could add additional control layer to the configuration such as 
- `AcceptNonStdTxs: true,` or `AcceptNonStdTxs: false,`
-  and `NonStdTxs:[]types.TxType` slice 
- by doing this we refuse the configuration of the standard txs 
- BTW also avoid the possibility of an empty configuration since the standard txs are not configurable.

Reply:
https://github.com/Qitmeer/qitmeer/pull/484/commits/bd8fc93eb32edb83e334030e0d3e84c1c4080aeb